libraries/AP_BattMonitor/AP_BattMonitor_INA2xx.h

```diff
@@ -3,17 +3,22 @@
 #include <AP_Common/AP_Common.h>
 #include <AP_HAL/I2CDevice.h>
 #include "AP_BattMonitor_Backend.h"
+#include <AP_Param/AP_Param.h>
 #include <utility>
 
-#define HAL_BATTMON_INA231_ENABLED defined(HAL_BATTMON_INA231_BUS) && defined(HAL_BATTMON_INA231_ADDR)
+#ifndef HAL_BATTMON_INA2XX_ENABLED
+#define HAL_BATTMON_INA2XX_ENABLED (BOARD_FLASH_SIZE > 1024)
```

Comment:
why not use HAL_MINIMIZE_FEATURE

Reply:
we're trying to avoid HAL_MINIMIZE_FEATURE, in favour of flash size for these new features. A lot of 1M boards don't have MINIMIZE_FEATURE set, but don't really want these types of features